rtengine/improccoordinator.h

```diff
@@ -222,6 +222,8 @@ class ImProcCoordinator : public StagedImageProcessor
 
     ImProcFunctions ipf;
 
+    Coord2D translateCoord(int x, int y);
```

Comment:
As far s I can see from the diff, this could be moved to an anonymous namespace in the implementation.

Reply:
Done, but i've used the `using namespace` trick inside the anonymous ns, to avoid repeating `rtengine::` all over the place. It seems to work, but i don't know, is it a bad practice? Thanks, sorry for my ignorance :-)

Reply:
Hmm, you could lower the scope to the function, but otherwise it's okay. Personally, I try to avoid `using namespace` as you import the whole thing, but YMMV.

Reply:
Oh, i didn't know you can also "use" a single class instead of the whole namespace... my ignorance again :-D. Fixed, should be better now